ckanext/sso/plugin.py

```diff
@@ -1,28 +1,27 @@
 # -*- coding: utf-8 -*-
 
-from __future__ import unicode_literals
-
 import logging
-import helper
+from . import helper
 
 from ckan import plugins
 from ckan.plugins import toolkit
 from pylons import config
```

Comment:
Did you succeed to use it with python3 ?
I have an issue with Pylon that is not compatible with python3
```
ckan  |   File "/usr/lib/python3.10/site-packages/pylons/controllers/core.py", line 108
ckan  |     except HTTPException, httpe:
ckan  |            ^^^^^^^^^^^^^^^^^^^^
ckan  | SyntaxError: multiple exception types must be parenthesized
```

Pylon is not in. the requirement file and the import is useless and could be replace in `helper.py` and `plugin.py` by the toolkit one:
```diff
< from pylons import config
---
> from ckan.plugins.toolkit import config
```